src/main/java/teammates/logic/api/EmailGenerator.java

```diff
@@ -399,6 +401,88 @@ private EmailWrapper generateFeedbackSessionEmailBaseForInstructorReminders(
                 EmailType.FEEDBACK_UNPUBLISHED.getSubject());
     }
 
+    public EmailWrapper generateFeedbackSessionResendLinksEmail(String userEmail) {
+        Instant endTime = new Date().toInstant();
+        Calendar calendar = Calendar.getInstance();
+        calendar.add(Calendar.MONTH, -6);
+        Instant startTime = calendar.getTime().toInstant();
```

Comment:
We are no longer using the `Date` and `Calendar` APIs.
```java
Instant endTime = Instant.now();
Instant startTime = endTime.minus(Period.ofMonths(6));

Reply:
Java throws the following exception when I use these: `java.time.temporal.UnsupportedTemporalTypeException: Unsupported unit: Months`.
![image](https://user-images.githubusercontent.com/28507005/39343472-879d8d14-4a0f-11e8-9a4c-f575a4a3adcc.png)
May I know if I have done anything wrong?

Reply:
It works when I use `Duration` instead of `Period`. However, `Duration` does not have `ofMonth` method, may I know if it is fine for me to `minus(Duration.ofDays(180))`?